cmd/skywire-cli/commands/hv/hvdmsg/hvdmsg.go

```diff
@@ -0,0 +1,90 @@
+package hvdmsg
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+	"github.com/toqueteos/webbrowser"
+
+	"github.com/skycoin/skywire/pkg/visor/visorconfig"
+)
+
+var (
+	path string
+	pkg  bool
+	pk   string
+	url  string
+)
+
+func init() {
+	RootCmd.AddCommand(dmsgUICmd)
+	dmsgUICmd.Flags().StringVarP(&path, "input", "i", "", "read from specified config file")
```

Comment:
I dont see this flag used yet in the commands.

Reply:
![image](https://user-images.githubusercontent.com/36607567/176554631-82c37857-c598-4e0e-8ce6-a35f88ee4cfc.png)